cli/tests/fixtures/extensions/permissions/correct-run-perms/PhylumExt.toml

```diff
@@ -0,0 +1,6 @@
+name = "correct-run-perms"
+description = "An extension with correctly specified permissions"
+entry_point = "main.ts"
```

Comment:
Just a curious idea, we could default to `main.ts`, right? Similar to how Rust loads `src/main.rs` by default. Completely unrelated to this PR, sorry.